src/python/src/grpc/_links/_transmission_test.py

```diff
@@ -94,7 +94,12 @@ def create_service_completion(self):
     return _intermediary_low.Code.OK, 'An exuberant test "details" message!'
 
   def assertMetadataEqual(self, original_metadata, transmitted_metadata):
```

Comment:
The "equal" part of this method name is now a falsehood and will have to be changed. How about "assertMetadataTransmitted"?